scania-digital-design-system/sdds
# Fix issue with range slider breaking storybook

**Describe pull-request**  
_Describe what the pull-request is about_
Fixes issue with storybook not working on divider,icons and card components 
Range controls needs to be a number and not a string

**How to test**  
_Add description how to test if possible_
1. Go to storybook
2. Check in cards, divider and icons so they work
